exporters/committime/collector_base.py

```diff
@@ -273,3 +400,110 @@ def _get_repo_from_build_config(self, build):
                     return git_uri + ".git"
 
         return None
+
+    def _find_git_clone_task_name(
+        self, pipeline_name: PipelineName, namespace: str
+    ) -> Optional[str]:
+        """
+        Get the Pipeline associated with a given PipelineRun,
+        and find the task name for the git-clone ClusterTask, if one exists.
+        Used to find the TaskRun that did the git checkout.
+        """
+        pipeline_resource = self._kube_client.resources.get(
+            api_version="tekton.dev/v1beta1", kind="Pipeline"
```

Comment:
As in previous comment. I do think such api_versions should be hidden in the function body.